test/integration/ipamd/ipamd_suite_test.go

```diff
@@ -51,13 +57,14 @@ var _ = BeforeSuite(func() {
 	numOfNodes = len(nodeList.Items)
 	Expect(numOfNodes).Should(BeNumerically(">", 1))
 
-	// Nominate the first node as the 'primary' node and force coredns deployment on this
+	// Nominate the first node as the one to run coredns deployment against
 	By("adding nodeSelector in coredns deployment to be scheduled on single node")
-	instanceID := k8sUtils.GetInstanceIDFromNode(nodeList.Items[0])
+	primaryNode := nodeList.Items[0]
```

Comment:
But if we have windows and linux nodes, we might end up picking windows node right?

Reply:
`nodeList` has a filter on linux nodes above, the `f.Options.NgNameLabelKey` and `f.Options.NgNameLabelVal`